extensions/contract-termination/src/main/resources/META-INF/services/org.eclipse.edc.spi.system.ServiceExtension

```diff
@@ -0,0 +1,14 @@
+#
+#  Copyright (c) 2024 sovity GmbH
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0
+#
+#  SPDX-License-Identifier: Apache-2.0
+#
+#  Contributors:
+#       sovity GmbH - initial API and implementation
+#
+
```

Comment:
we've never done this. we don't valiidate it, let's not start with it if it's not necessary